R/teal_slices-store.R

```diff
@@ -75,6 +75,13 @@ slices_restore <- function(file) {
             } else {
               slice[[field]]
             }
+        } else {
+          slice[[field]] <-
+            if (field == "selected") {
+              character(0)
+            } else {
+              NULL
+            }
```

Comment:
:heart: the simplicity of solution. 1 line suggestion, but feel free to ignore

```suggestion
          slice[[field]] <- switch(field, selected = character(0), NULL)
```

Reply:
Don't you mean `slice[[field]] <- switch(field, "selected" = character(0), NULL)` :thinking: